jinja2/403.html

```diff
@@ -23,8 +23,6 @@ <h1 class="page-title">{% block page_title %}{{ _('Permission Denied') }}{% endb
       <p class="notice">{{ _('The wiki is in read-only mode.') }}</p>
     {% elif reason == 'kumabanned' %}
       <p class="notice">{{ _('Your profile has been banned from making edits.') }}</p>
-    {% elif reason == 'bannedprofile' %}
-      <p class="notice">{{ _("This user's profile has been banned.") }}</p>
```

Comment:
This notice should be moved to the 404 handler, if possible with the same string so that translators don't have to re-translate.